src/utils.ts

```diff
@@ -125,6 +125,20 @@ export const compareText = (actual: string, expected: string | RegExp, { ignoreC
     if (trim) {
         actual = actual.trim()
     }
+    if (typeof replace === 'object') {
+        const replacers = Array.isArray(replace)
+            ? [replace]
+            : Object.values(replace)
+
+        if (replacers.some(r => r.length !== 2)) {
+            throw new Error('Replacers need to have a searchValue and a replaceValue');
+        }
+
+        for (const replacer of replacers) {
+            const [searchValue, replaceValue] = replacer
+            actual = actual.replace(searchValue, replaceValue as string)
+        }
+    }
```

Comment:
This seems to be very similar to what happens in line 200, maybe make a utility function of it?

Reply:
Sure thing